python/cudf/cudf/tests/test_groupby.py

```diff
@@ -392,19 +392,72 @@ def emulate(df):
 
 
 @pytest.fixture(scope="module")
-def groupby_jit_data():
-    np.random.seed(0)
+def groupby_jit_data_small():
+    """
+    Small dataset for testing groupby apply with jit
```

Comment:
Can we write this comment with complete sentences / capitalization? It also feels like it's wrapped pretty harshly, like 50-60 chars / line? Same request for other comments below.